giskard/core/suite.py

```diff
@@ -311,22 +311,28 @@ class Suite:
             A mapping of suite parameters with their corresponding SuiteInput objects.
         name : str
             A string representing the name of the suite.
+        default_params : Dict[str, Any]
+            A dictionary containing the default parameters for the tests in the suite.
     """
 
     id: int
     tests: List[TestPartial]
     name: str
+    default_params: Dict[str, Any]
 
-    def __init__(self, name=None) -> None:
+    def __init__(self, name=None, **default_params) -> None:
```

Comment:
Can we use a normal argument instead of the catch-all `**kwargs`?

Reply:
Sure, it's done.

I realize that the parameters like `name=""` will be ignored otherwise.